manifests/compute/foreman_compute.pp

```diff
@@ -1,7 +1,8 @@
-# Provides a common class to install foreman-compute
-# used by multiple compute resources
+# = Provides a common class to install foreman-compute
```

Comment:
Actually, this class can also be killed. Since https://github.com/theforeman/foreman-packaging/commit/88e977a58d26350b613ec68b86c93eca17d8a941 (Foreman 1.22) `foreman-compute` no longer exists.